lib/modules/filteReddit.js

```diff
@@ -590,9 +608,83 @@ export function toggleFilter(subreddit: string): boolean {
 
 	Options.set(module, 'subreddits', filteredReddits);
 
+	if (module.options.useRedditFilters.value) reconcileNativeFilters();
+
 	return (subredditIndex === -1);
 }
 
+async function reconcileNativeFilters() {
+	const user = loggedInUser();
+
+	if (!user) {
+		Notifications.showNotification({
```

Comment:
IMO "I can't do that, Dave, you're not logged in" should only show if the user tries to `forceSyncFilters`, since many people browse logged-out all the time.